build-script-helper.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Does this script run in the CI tests for pull requests?

(Just making sure so we can ensure that whatever tool using this script actually does have python 3 installed and there are no syntax issues with this script when running with 3 vs 2.)